inst/rmarkdown/templates/visc_empty/skeleton/skeleton.Rmd

```diff
@@ -48,7 +48,11 @@ install_load_cran_packages(packages_needed)
 # knitr options
 opts_chunk$set(cache = FALSE, message = TRUE, warning = TRUE, echo = FALSE, 
                dev = c("png", "pdf"), dpi = 200, out.width = '100%', 
-               out.extra = '', fig.align = "center", fig.pos = "H")
+               out.extra = '', fig.pos = "H")
+# fig.align argument is not supported in Word (align in template docx)
```

Comment:
wondering if it would be better to move up `output_type <- get_output_type()`, which is currently two chunks below this, and then just use `if (output_type == 'latex')` instead?

Reply:
Maybe! Will need to think about how this switch works here:

```
> get_output_type
function() {

  current_output_type <- knitr::opts_knit$get('rmarkdown.pandoc.to')

  # if interactive, set to pandoc for easier visualization
  ifelse(!is.null(current_output_type) && current_output_type == 'latex',
         'latex', 'pandoc')

}
<environment: namespace:VISCtemplates>
```

Reply:
It seems like `get_output_type()` helps choose how things are visualized depending on whether the session is interactive or not, whereas what I'm trying to do is to use different knitr arguments for the whole Rmd depending on if we're doing PDF or Word.  So I'm thinking re-using `output_type` might not be the best thing to do there.

Reply:
ah ok I see. I wasn't understanding the scope of `get_output_type()` properly. thanks for clarifying!